presentation/src/main/java/com/london/presentation/screen/moiveDetalis/MovieDetailsScreen.kt

```diff
@@ -0,0 +1,523 @@
+                hasIcon = false,
+                isLoading = false,
+                isDisabled = state.movieHaveTrailer.not(),
+                icon = null,
+                modifier = Modifier.weight(1f)
+            )
+        }
+    }
+}
+
+@Composable
+private fun RatingAndMetaRow(
+    rate: String,
+    time: String,
+    date: String,
+) {
+    Row(
+        horizontalArrangement = Arrangement.spacedBy(4.dp),
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        IconWithText(
+            icon = drawable.star,
+            contentDesc = stringResource(star),
+            tint = NovixTheme.colors.yellowAccent,
+            text = rate,
+            textColor = NovixTheme.colors.title
+        )
+        Dot()
+        IconWithText(
+            icon = drawable.time_04,
+            contentDesc = stringResource(time_icon),
+            tint = NovixTheme.colors.body,
+            text = time,
+            textColor = NovixTheme.colors.body
+        )
+        Dot()
+        IconWithText(
+            icon = drawable.calendar_03,
+            contentDesc = stringResource(calendar),
+            tint = NovixTheme.colors.body,
+            text = date,
+            textColor = NovixTheme.colors.body
+        )
+    }
+}
+
+@Composable
+private fun IconWithText(
+    icon: Int, contentDesc: String, tint: Color, text: String, textColor: Color
+) {
+    Icon(
+        painter = painterResource(icon), contentDescription = contentDesc, tint = tint
+    )
+    Text(
+        text, style = NovixTheme.typography.label.small, color = textColor
+    )
+}
+
+@Composable
+private fun GenreRow(genres: List<String>) {
+    Row(
+        horizontalArrangement = Arrangement.spacedBy(8.dp),
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        genres.forEachIndexed { index, genre ->
+            Text(
+                genre, style = NovixTheme.typography.label.small, color = NovixTheme.colors.body
+            )
+            if (index != genres.lastIndex) Icon(
+                painter = painterResource(drawable.ellipse_2),
+                contentDescription = stringResource(separator),
+                tint = NovixTheme.colors.hint
+            )
+        }
+    }
+}
+
+@Composable
+private fun Dot(modifier: Modifier = Modifier) {
+    Icon(
+        painter = painterResource(drawable.ellipse_2),
+        contentDescription = stringResource(dot),
+        tint = NovixTheme.colors.body,
+        modifier = modifier
+    )
+}
+
+@Composable
+private fun MovieDetailsImage(
+    images: List<Any>,
+    onSaveClick: () -> Unit,
+    modifier: Modifier = Modifier,
+    isSaved: Boolean = false,
+    imageDescription: String? = null,
+    loadingState: MutableState<Boolean> = remember { mutableStateOf(false) },
+    currentImageIndex: Int,
+    direction: Int,
+    onBackClick: () -> Unit
+) {
+    Box(
+        modifier = modifier
+            .fillMaxWidth()
+            .aspectRatio(160f / 100f)
```

Comment:
you can use either `16:10` or `8:5`.
https://en.wikipedia.org/wiki/16:10_aspect_ratio
```suggestion
            .aspectRatio(8f / 5f)
```